apps/condo/domains/notification/utils/serverSchema/index.js

```diff
@@ -11,8 +11,9 @@ const { find } = require('@open-condo/keystone/schema')
 const { extractReqLocale } = require('@open-condo/locales/extractReqLocale')
 
 const { LOCALES } = require('@condo/domains/common/constants/locale')
+const { MessageAppBlackList } = require('@condo/domains/miniapp/utils/serverSchema')
```

Comment:
Notification is a top-level domain. You should not use import from any other domain here